apps/protailwind/src/trpc/routers/module-progress.ts

```diff
@@ -22,20 +20,12 @@ export const moduleProgressRouter = router({
         return null
       }
 
-      const getModuleByType = async (slug: string, type?: string) => {
-        switch (type) {
-          case 'tutorial':
-            return await getTutorial(slug)
-          default:
-            return await getWorkshop(slug)
-        }
-      }
-
-      const module = await getModuleByType(input.slug, input.type)
+      const module = await getModule(input.slug)
```

Comment:
we can just do this. I've added sections to groq query so that it resolves correctly.